examples/testapp/src/pages/auto-sub-account/index.page.tsx

```diff
@@ -261,6 +269,52 @@ export default function AutoSubAccount() {
     }
   };
 
+  const handleUsdcSend = async (amount: string) => {
+    if (!provider || accounts.length < 2) return;
+
+    try {
+      setSendingUsdcAmounts((prev) => ({ ...prev, [amount]: true }));
+      const usdcAddress = '0x036cbd53842c5426634e7929541ec2318f3dcf7e';
+      const to = '0x8d25687829d6b85d9e0020b8c89e3ca24de20a89';
+      const value = parseUnits(amount, 6); // USDC has 6 decimals
+
+      // Encode ERC20 transfer function call
+      const data = encodeFunctionData({
+        abi: [
+          {
+            name: 'transfer',
+            type: 'function',
+            inputs: [
+              { name: 'to', type: 'address' },
+              { name: 'amount', type: 'uint256' },
+            ],
+            outputs: [{ name: '', type: 'bool' }],
+          },
+        ],
+        functionName: 'transfer',
+        args: [to, value],
+      });
+
+      const response = await provider.request({
+        method: 'eth_sendTransaction',
+        params: [
+          {
+            from: accounts[0], // Use sub account (accounts[1])
```

Comment:
this comment is confusing given what the code actually says. accounts[0] is the universal, so why reference sub account?

Reply:
true. it actually depends because in auto sub accounts mode (which is what i've been testing with) sub account is index 0 and in normal mode universal is index 0

will remove that comment